druid-shell/Cargo.toml

```diff
@@ -16,7 +16,7 @@ use_gtk = ["gtk", "gtk-sys", "gio", "gdk", "gdk-sys", "glib", "glib-sys", "cairo
 default-target = "x86_64-pc-windows-msvc"
 
 [dependencies]
-piet-common = "0.0.11"
+piet-common = {version = "0.0.11", features = ["png"]}
```

Comment:
Given that this is all testing code, I think the `png` feature should only be enabled for testing via `[dev-dependencies]` or something like that.